config/all/roles.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+/**
+ * @fileoverview Roles configuration
+ *
+ * @returns {array} Array of user roles
+ */
+module.exports = [
+  {
+    name: 'Administrator',
+    id: 'admin',
+    allows: [
+      {
+        resources: '/content/services',
```

Comment:
@Snugug There are other ways to do this, but was getting very heavy for this PR. Ok if I make a story for that?